### PR TITLE
25). Add topic files 71 to 74

### PR DESCRIPTION
**Pull request by jameshkramer**
This PR is for user story #1576037.

It adds four article files, makes corresponding deletions from the C# and VB folders, and adds redirection entries. It also fixes toc.yml.

Please merge #17846, #17892, and #17928 before merging this.

cc: @mairaw @tfosmark @marteeleigh

Contributes to #4728

### CODE REVIEW

File: .openpublishing.redirection.json [hunk 1]
```diff
@@ -75,7 +75,7 @@
     },
     {
       "source_path": "docs/architecture/modernize-with-azure-containers/lift-and-shift-existing-apps-devops/when-to-deploy-windows-containers-to-azure-container-service-kubernetes.md",
-      "redirect_url": "/dotnet/architecture/modernize-with-azure-containers//modernize-existing-apps-to-cloud-optimized/when-to-deploy-windows-containers-to-azure-container-service-kubernetes"
+      "redirect_url": "/dotnet/architecture/modernize-with-azure-containers/modernize-existing-apps-to-cloud-optimized/when-to-deploy-windows-containers-to-azure-container-service-kubernetes"
```
Comment (mairaw) [inline .openpublishing.redirection.json:78]:
```suggestion
      "redirect_url": "/dotnet/architecture/modernize-with-azure-containers//modernize-existing-apps-to-cloud-optimized/when-to-deploy-windows-containers-to-azure-container-service-kubernetes"
```

File: .openpublishing.redirection.json [hunk 2]
```diff
@@ -2326,7 +2346,7 @@
     },
     {
       "source_path": "docs/framework/migration-guide/mitigation-claimsidentity-constructor.md",
-      "redirect_url": "/dotnet/framework/migration-guide//retargeting/4.6.1-4.6.2"
+      "redirect_url": "/dotnet/framework/migration-guide/retargeting/4.6.1-4.6.2"
```
Comment (mairaw) [inline .openpublishing.redirection.json:2349]:
```suggestion
      "redirect_url": "/dotnet/framework/migration-guide//retargeting/4.6.1-4.6.2"
```

File: docs/standard/linq/do-streaming-transformations-text-xml.md [hunk 3]
```diff
@@ -60,7 +67,7 @@ class Program
 }
 ```
 
- This example produces the following output:
+The example produces this output:
```
Comment (mairaw) [inline docs/standard/linq/do-streaming-transformations-text-xml.md:70]:
```suggestion
The example produces the following output:
```

File: docs/standard/linq/do-streaming-transformations-text-xml.md [hunk 4]
```diff
@@ -1,17 +1,24 @@
 ---
```
Comment (mairaw) [inline docs/standard/linq/do-streaming-transformations-text-xml.md:1]: I'd probably remove do from the file name here

File: docs/standard/linq/do-streaming-transformations-text-xml.md [hunk 5]
```diff
@@ -1,17 +1,24 @@
...
 
-One approach to processing a text file is to write an extension method that streams the text file a line at a time using the `yield return` construct. You then can write a LINQ query that processes the text file in a lazy deferred fashion. If you then use <xref:System.Xml.Linq.XStreamingElement> to stream output, you then can create a transformation from the text file to XML that uses a minimal amount of memory, regardless of the size of the source text file.
+# How to do streaming transformations of text to XML in C# (LINQ to XML)
 
- There are some caveats regarding streaming transformations. A streaming transformation is best applied in situations where you can process the entire file once, and if you can process the lines in the order that they occur in the source document. If you have to process the file more than once, or if you have to sort the lines before you can process them, you will lose many of the benefits of using a streaming technique.
+You can use an extension method that releases a line at a time to stream a text file for processing. This technique reduces memory requirements compared to techniques which load the entire file and then process it.
```
Comment (mairaw) [inline docs/standard/linq/do-streaming-transformations-text-xml.md:12]: @BillWagner do you wanna review the text changes here?

File: docs/standard/linq/list-all-nodes-tree.md [hunk 6]
```diff
@@ -1,22 +1,29 @@
...
+  - "vb"
 ms.assetid: 3e934371-f4c6-458b-9f6b-f9061b596f5b
 ---
-# How to list all nodes in a tree (C#)
 
-Sometimes it is helpful to list all nodes in a tree. This can be useful when learning exactly how a method or property affects the tree. One approach to listing all nodes in a textual form is to generate an XPath expression that exactly and specifically identifies any node in the tree.
+# How to list all nodes in a tree (LINQ to XML)
 
-It is not particularly helpful to execute XPath expressions using [!INCLUDE[sqltecxlinq](~/includes/sqltecxlinq-md.md)]. XPath expressions have poorer performance than [!INCLUDE[sqltecxlinq](~/includes/sqltecxlinq-md.md)] queries, and [!INCLUDE[sqltecxlinq](~/includes/sqltecxlinq-md.md)] queries are much more powerful. However, as a way to identify nodes in the XML tree, XPath works well.
+Sometimes it is helpful to list all nodes in a tree – for example, to learn exactly how a method or property affects the tree. The example below for C# and Visual Basic lists the nodes of a tree. Each node is represented by an XPath expression that specifies the location of the node in the tree.
```
Comment (mairaw) [inline docs/standard/linq/list-all-nodes-tree.md:13]:
```suggestion
Sometimes, it's helpful to list all nodes in a tree – for example, to learn exactly how a method or property affects the tree. The following example for C# and Visual Basic lists the nodes of a tree. Each node is represented by an XPath expression that specifies the location of the node in the tree.
```

File: docs/standard/linq/modify-office-open-xml-document.md [hunk 7]
```diff
@@ -0,0 +1,270 @@
...
+dev_langs:
+  - "csharp"
+  - "vb"
+ms.assetid: 467d489c-2b1b-453b-a757-8ac180e82a96
+---
+# How to modify an Office Open XML document (LINQ to XML)
+
+This article presents an example for C# and Visual Basic that opens an Office Open XML document, modifies it, and saves it.
+
+For more information on Office Open XML, see [Open XML SDK](https://github.com/OfficeDev/Open-XML-SDK) and [www.ericwhite.com](http://ericwhite.com/).
```
Comment (mairaw) [inline docs/standard/linq/modify-office-open-xml-document.md:14]:
```suggestion
For more information about Office Open XML, see [Open XML SDK](https://github.com/OfficeDev/Open-XML-SDK) and [www.ericwhite.com](http://ericwhite.com/).
```

File: docs/standard/linq/retrieve-paragraphs-office-open-xml-document.md [hunk 8]
```diff
@@ -0,0 +1,316 @@
...
+  - "csharp"
+  - "vb"
+ms.assetid: cc2687cf-d648-451e-88ac-3847c6c967c8
+---
+
+# How to retrieve paragraphs from an Office Open XML document (LINQ to XML)
+
+This article provides an example that opens an Office Open XML document and retrieves a collection of all of the paragraphs in the document.
+
+For more information on Office Open XML, see [Open XML SDK](https://github.com/OfficeDev/Open-XML-SDK) and [Eric White's Blog](http://www.ericwhite.com/).
```
Comment (mairaw) [inline docs/standard/linq/retrieve-paragraphs-office-open-xml-document.md:15]:
```suggestion
For more information about Office Open XML, see [Open XML SDK](https://github.com/OfficeDev/Open-XML-SDK) and [Eric White's Blog](http://www.ericwhite.com/).
```

**Review by mairaw — COMMENTED** [review]
LGTM. Left some comments to be addressed before merging.

**Comment by mairaw** [comment]
@jameshkramer ping me when comments and merge conflicts are resolved. Thanks!

**Comment by mairaw** [comment]
@jameshkramer do you wanna resolve the merge conflicts here?

**Review by mairaw — APPROVED** [review]
LGTM. @BillWagner do you want to take a look at the changes he's made? Thanks.

**Review by BillWagner — APPROVED** [review]
This LGTM. I'll :shipit: now.

**Pull request merged**